apps/dapp-console/app/settings/contracts/AddContractForm.tsx

```diff
@@ -118,7 +152,7 @@ export const AddContractForm = ({
       <div className="mt-3 flex flex-row justify-end w-full">
         <Button
           disabled={!form.formState.isValid}
```

Comment:
nit: feel free to add in follow up pr but should there be a loading state on the button?

Reply:
yup will do! i haven't added any loading states yet be will when i start to add them once the backend integration is done